client/public/index.html

```diff
@@ -15,6 +15,10 @@
 
     <title>React App</title>
     <link rel="stylesheet" href="styles.css" />
+    <link
```

Comment:
I love Source Code Pro, i seriously do.
But I tried Raleway font and it looks much better!
Could you maybe add that? Think it looks much clear and not like a typed font...
Would be providing the link for the same

Reply:
See it for yourself :wink:
![image](https://user-images.githubusercontent.com/43881544/85264421-86686680-b425-11ea-9c69-c42858161e30.png)